plugins/inputs/vsphere/README.md

```diff
@@ -168,15 +171,30 @@ For example, to disable collection of VMs, add this:
 
 ### Objects and Metrics Per Query
 
-Default settings for vCenter 6.5 and above is 256. Prior versions of vCenter have this set to 64. A vCenter administrator
-can change this setting, which should be reflected in this plugin. See this [VMware KB article](https://kb.vmware.com/s/article/2107096)
-for more information.
+By default, in vCenter's configuration, a limit is set to the number of entities that are included in a performance chart query. Default settings for vCenter 6.5 and above is 256. Prior versions of vCenter have this set to 64. 
+A vCenter administrator can change this setting, see this [VMware KB article](https://kb.vmware.com/s/article/2107096) for more information.
+
+Any modification should be reflected in this plugin by modifying the parameter `max_query_objects`
+
+```
+  ## number of objects to retreive per query for realtime resources (vms and hosts)
+  ## set to 64 for vCenter 5.5 and 6.0 (default: 256)
+  # max_query_objects = 256
```

Comment:
I don't think we should have the toml examples, since they are already shown above in the config.